scripts/_env_var_util.sh

```diff
@@ -61,18 +94,17 @@ _env_var_util::validate () {
         valid=false
     } >&2
 
-    {
-        [ -z "${NETWORK_CONFIG}" ] \
-        || [ "${NETWORK_CONFIG}" == 'default_network' ] \
-        || [ "${NETWORK_CONFIG}" == 'new_network' ] \
-        || [ "${NETWORK_CONFIG}" == 'multi_nic_network' ];
-    } || {
-        echo "NETWORK_CONFIG='${NETWORK_CONFIG}'"
-        echo "  - Must be one of ['default_network', 'new_network', 'multi_nic_network']."
-        valid=false
-    } >&2
+    declare -ar expected_network_configs=('default_network' 'new_network' 'multi_nic_network')
+    [ -z "${NETWORK_CONFIG}" ] \
+        || _env_var_util::expect_contains expected_network_configs NETWORK_CONFIG \
+        || valid=false
```

Comment:
This asserts that `NETWORK_CONFIG` is either an empty string or one of those three values. If it isn't, then the function will return an error (at the end so it can print ALL the problems in the environment in one run)